--- gocr-0.3.2/src/Makefile.in.orig	Thu May 24 15:05:31 2001
+++ gocr-0.3.2/src/Makefile.in	Wed Jul  4 01:42:05 2001
@@ -61,9 +61,10 @@
 # PHONY = don't look at file clean, -rm = start rm and ignore errors
 .PHONY : clean proper install
 install: all
-	$(INSTALL) ../bin/$(PROGRAM) $(bindir)
-	$(INSTALL) $(LIBPGMASCLIB) $(libdir)
-	$(INSTALL) $(INCLUDEFILES) $(includedir)
+	$(INSTALL) -d $(DESTDIR)$(bindir) $(DESTDIR)$(libdir) $(DESTDIR)$(includedir)
+	$(INSTALL) ../bin/$(PROGRAM) $(DESTDIR)$(bindir)
+	$(INSTALL) $(LIBPGMASCLIB) $(DESTDIR)$(libdir)
+	$(INSTALL) $(INCLUDEFILES) $(DESTDIR)$(includedir)
 clean:
 	-rm -f *.o *~
 
--- gocr-0.3.2/man/Makefile.in.orig	Thu May 24 15:05:31 2001
+++ gocr-0.3.2/man/Makefile.in	Wed Jul  4 01:42:59 2001
@@ -15,7 +15,8 @@
 # PHONY = don't look at file clean, -rm = start rm and ignore errors
 .PHONY : clean proper install
 install: all
-	$(INSTALL) ../man/man1/$(PROGRAM).1 $(mandir)/man1
+	$(INSTALL) -d $(DESTDIR)$(mandir)/man1
+	$(INSTALL) ../man/man1/$(PROGRAM).1 $(DESTDIR)$(mandir)/man1
 
 clean:
 	-rm -f *.o *~
