--- gocr-0.3.6/frontend/gnome/src/functions.c~	Sun May 12 14:36:29 2002
+++ gocr-0.3.6/frontend/gnome/src/functions.c	Sun May 12 14:36:29 2002
@@ -209,7 +209,7 @@
 			"DUST=%d\n"
 			"SPACE=%d\n"
 			"VIEWER=%s\n"
-			"PATH=/usr/local/bin/gocr\n",
+			"PATH=/usr/bin/gocr\n",
 			nGrey,
 			nDust,
 			nSpace,
@@ -334,8 +334,8 @@
   gtk_widget_show ( GTK_WIDGET (label));
   if ( Path == NULL)
     {
-      Path = g_malloc(strlen("/usr/local/bin/gocr\0"+1));
-      sprintf(Path,"/usr/local/bin/gocr");
+      Path = g_malloc(strlen("/usr/bin/gocr\0"+1));
+      sprintf(Path,"/usr/bin/gocr");
     }
 
   TxtPath=gtk_entry_new();
